src/app/forms/components/dynamic-form-field/dynamic-form-field.component.html

```diff
@@ -1,7 +1,9 @@
-<ng-container
+<div
   *ngIf="form && control && control.value.meta.editType !== formNodeEditTypes.HIDDEN"
   [ngSwitch]="control.value.meta.editType"
   [formGroup]="form"
+  appNestingLevel
+  [nestingLevel]="$any(form).meta.nestingLevel ?? 0"
```

Comment:
You can simultaneously apply the directive and nesting level at once if you rename input property to appNestingLevel within the directive

Reply:
Didn't know that, thanks. Will modify.